pkg/rhsm/secrets.go

```diff
@@ -42,18 +42,25 @@ type ConsumerSecrets struct {
 }
 
 func getRHSMSecrets() (*RHSMSecrets, error) {
-	keys, err := filepath.Glob("/etc/pki/entitlement/*-key.pem")
-	if err != nil {
-		return nil, err
+	globs := []string{
```

Comment:
Maybe worth a comment here that the order is deliberate, something like: `// search /etc/ first to allow container users to override the entitlements` or something